src/test/tests/databases/multi_format.py

```diff
@@ -54,8 +54,8 @@
 AddPlot("FilledBoundary", "material(mesh)")
 DrawPlots()
 Test("multi_format_02")
-
-SetTimeSliderState(TimeSliderGetNStates() / 2)
+# note: py3 div creates float
```

Comment:
Don't think this comment is necessary everywhere this cast is now needed. It works either way and is arguably *more correct* now than before without explanation.

Reply:
more comments == more helpful in the future.  Python 2 folks may recall old behavior.